ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java

```diff
@@ -74,6 +261,9 @@ public void setText(ReactTextUpdate update) {
         setBreakStrategy(update.getTextBreakStrategy());
       }
     }
+
+    // Ensure onLayout is called so the inline views can be repositioned.
```

Comment:
As I said previously, I don't think we need this line, but it certainly doesn't hurt.